Content.Shared/Access/IdCardComponent.cs

```diff
@@ -1,13 +1,11 @@
-using Content.Server.Access.Systems;
-using Content.Server.PDA;
-using Robust.Shared.Analyzers;
 using Robust.Shared.GameObjects;
 using Robust.Shared.Serialization.Manager.Attributes;
 
-namespace Content.Server.Access.Components
+namespace Content.Shared.Access
 {
+    // TODO BUI NETWORKING if ever clients can open their own BUI's (id card console, pda), then this data should be
+    // networked.
     [RegisterComponent]
-    [Friend(typeof(IdCardSystem), typeof(PDASystem))]
```

Comment:
So there is a `SharedPDASystem` but no `SharedIdCardSystem` (only serverside atm). So as far as I know there is no way to specify friends without either creating a `SharedIdCardSystem`, or splitting the `IdCardComponent` into shared & client/server versions.
 
if there's a way around that let me know, but I don't really want to create those classes solely for the purpose of friendship declarations. Eventually I want to move access to shared anyways, so that opening & closing doors can be predicted, and I can just make them, friends again at that point.

Reply:
Actually never mind, doors & such requires sharing the `AccessComponent` and `AccessReader` system, not the id card system.

I could still just move `IdCardSystem` to shared, but it it still feels weird to do it for no reason other than to have friendship declarations. With access & Id seperate, I can't think of a reason why the client would need to care about the id card system.

Reply:
The only reason `IdCardComponent` was moved to shared in the first place was to handle entity whitelists. It would be nice if there was just a for clients to handle entity whitelists that rely on server-side components. One option would be to just add a "This has a <server-side component>" tag to entities and use that for the whitelist instead, but that feels icky.

I asked on discord whether I should add a tag or just make the component shared and the conclusion was just make it shared.

Reply:
Just make an empty shared system. That's what we've been doing in other PRs when they're needed.

Reply:
done.